### ISSUE TITLE
api for unreliable/unordered send

### ISSUE THREAD

**Issue by doongjohn** [treeform/netty#26]
In some situations, the reliable send function is not required. Is there a way to do an unreliable/unordered send?

**Comment by treeform** [comment]
No, sorry, there isn't. Once you have a reliable channel, it is simpler to send everything reliably. Feel free to fork and add your own unreliable/unordered send if that is better for your use case.
